README.md

```diff
@@ -1,9 +1,38 @@
-# User Login.
-This library is for user login with active directory account.
+ # User Login.
+This package provides login functionality for authenticating users with Active Directory accounts.
 
 Built for [Laravel 11](https://laravel.com/).
 
 ## Installation
 
-Add the library via Composer: `composer require networkrailbusinesssystems/user-login`
+Add the library using Composer:
+   ```
```

Comment:
Do not indent Markdown files like this.

Everything should be left-aligned.

Reply:
fixed